xbmc/cores/EdlEdit.h

```diff
@@ -8,6 +8,10 @@
 
 #pragma once
 
+#include <chrono>
+
+using namespace std::chrono_literals;
```

Comment:
Shouldn't use a `using` statement in a header.

Reply:
Can you brace initialize the timing members?

Reply:
That's something I did not think about before but the reason is definitely on your side. Thanks!